starter-code/models/post.js

```diff
@@ -0,0 +1,13 @@
+/*jshint esversion: 6 */
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+mongoose.connect('mongodb://localhost/linkedIn',  {useMongoClient: true});
+
+const PostSchema = new Schema({
+  content    : String,
+  creator    : String,
```

Comment:
creator should be the user's id and not the username

in which case, the right type for the field would be `creator: Mongoose.Schema.ObjectId`